### ISSUE TITLE
Fails to load in Playdate Simulator

### ISSUE THREAD

**Issue by mhughson** [mierau/gamekid#4]
Is it expected that gamekid cannot run in the Playdate Simulator?

```
11:01:20: Loading: C:\Users\Matt\Downloads\gamekid-0.3\gamekid.pdx
11:01:20: Loading: Failed
Load failed: /
```
![image](https://user-images.githubusercontent.com/310185/167916491-7989a36d-d830-48b8-8fab-eb45bf3a7538.png)

I'm not sure if there is more useful information or logs I could get. Just let me know.

**Comment by deanhuff** [comment]
I'm just guessing, but it looks like the emulator is compiled for arm64.  so you'd need a M1 mac to run in sim.  (maybe it would run on a rpi?)

**Comment by mierau** [comment]
Yeah, I've only focused on making this work smoothly on device. I have yet to explore getting a dll compiled for Windows and Intel Macs (I'm currently compiling the release on my M1 Mac). This is an interesting idea though for those who want to perhaps exclusively run it in the Playdate Simulator.
